apiserver/facades/client/bundle/mock_test.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package bundle_test
+
+import (
+	"github.com/juju/testing"
+)
+
+type mockState struct {
```

Comment:
A green-field situation like this would be a good chance to use GoMock rather than another of these hand-rolled stubs.